try:
    2*3
except TypeError:
    print("An exception was raised")
else:
    print("Oh yes! No exceptions were raised.")